File: 프로그래머스/lv1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.cs
using System;

public class Solution {
    public int[] solution(long n) {
        int[] answer = new int[] {};
        
        string tmp=n.ToString();
        int i=0,Len=tmp.Length;
        answer=new int[Len];
        
        while(n>0)
        {
            answer[i]=(int)(n%10);
            n/=10;
            if(n==0)
                break;
            i++;
        }
        return answer;
    }
}

File: 프로그래머스/lv1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.java
class Solution {
    public int[] solution(long n) {
        String str = Long.toString(n);
        String s = new StringBuilder(str).reverse().toString();
        char[] tmp = s.toCharArray();
        int[] answer = new int[tmp.length];
        
        for(int i = 0; i < answer.length; i++){
           answer[i] = tmp[i] -'0';
        }
        return answer;
    }
}